src/bundle/Resources/public/scss/mixins/_drag-and-drop.scss

```diff
@@ -251,3 +251,23 @@
         display: none;
     }
 }
+
+@mixin empty-drop-zone {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: calculateRem(50px);
+
+    &__image {
+        margin-top: calculateRem(90px);
+    }
```

Comment:
I wonder: isn't this part too specific for general-purpose mixin?

Reply:
Discussed in private: it seems that in most places, if not in all, we will have an image in an empty drop zone.